### ISSUE TITLE
CSV download doesn't give all politicians, even though it sort of suggests it might

### ISSUE THREAD

**Issue by pmk01** [mysociety/theyworkforyou#1769]
I want a list of all the MLAs you've got.

When I navigate to https://www.theyworkforyou.com/mlas/ and select [Historical list of all MLAs](https://www.theyworkforyou.com/mlas/?all=1), I do indeed see such a list. But then when I "Download this list as a CSV suitable for Excel", I don't get the list in front of me, I get a CSV of only current MLAs.

(This is the case with all other assemblies).

Perhaps the link isn't supposed to give me everyone; if so, it would be useful to make this clear.

**Comment by ajparsons** [comment]
Technical problem aside for the mo - the information you're after should be in this spreadsheet here https://pages.mysociety.org/politician_data/downloads/uk-politician-data-simple-memberships-csv/latest

(Just need to filter down on Wales or the MS role)

**Comment by pmk01** [comment]
Yep, sorry, I should have said, I found that! It took me a while to work out which sheet I needed though, which is why I thought this would be easier.

**Comment by dracos** [comment]
FWIW if you combine the parameters from the two URLs - all=1&f=csv - I think that gives you the full list as CSV then.
